src/pickley/bstrap.py

```diff
@@ -160,8 +159,15 @@ def find_base(base):
     abort(f"Make sure '{candidates[0]}' is writeable.")
 
 
-def get_latest_version(package_name):
-    data = http_get(f"https://pypi.org/pypi/{package_name}/json")
+def get_latest_pickley_version(mirror):
+    if not mirror:
+        mirror = "https://pypi.org/simple"
+
+    # This is a temporary measure, eventuall we'll use `uv describe` for this
+    url = os.path.dirname(mirror)
```

Comment:
I believe the mirror can be set with a harmless trailing slash -- you probably want to rstrip those here